build.json

```diff
@@ -3,6 +3,9 @@
     "PYTHON_VERSION": {
       "default": "3.9.13"
     },
+    "PGSQL_VERSION": {
+      "default": "15"
```

Comment:
Do we need also to pin the minor version? We can intentionally make more restrictions to avoid confliction.

Reply:
Postgres guarantees that API won't change for the same major version. I am confident that fixing the major version is good enough.